src/data/cel.rs

```diff
@@ -470,36 +472,65 @@ mod tests {
 
     #[test]
     fn expressions_to_json_resolve() {
-        super::super::property::test::TEST_PROPERTY_VALUE.set(Some("true".bytes().collect()));
+        property::test::TEST_PROPERTY_VALUE.set(Some((
+            property::Path::new(vec![
+                "filter_state",
+                "wasm.kuadrant.auth.identity.anonymous",
+            ]),
+            "true".bytes().collect(),
+        )));
         let value = Expression::new("auth.identity.anonymous").unwrap().eval();
         assert_eq!(value, true.into());
-        super::super::property::test::TEST_PROPERTY_VALUE.set(Some("42".bytes().collect()));
+
+        property::test::TEST_PROPERTY_VALUE.set(Some((
+            property::Path::new(vec!["filter_state", "wasm.kuadrant.auth.identity.age"]),
+            "42".bytes().collect(),
+        )));
         let value = Expression::new("auth.identity.age").unwrap().eval();
         assert_eq!(value, 42.into());
-        super::super::property::test::TEST_PROPERTY_VALUE.set(Some("42.3".bytes().collect()));
+
+        property::test::TEST_PROPERTY_VALUE.set(Some((
+            property::Path::new(vec!["filter_state", "wasm.kuadrant.auth.identity.age"]),
+            "42.3".bytes().collect(),
+        )));
         let value = Expression::new("auth.identity.age").unwrap().eval();
         assert_eq!(value, 42.3.into());
-        super::super::property::test::TEST_PROPERTY_VALUE.set(Some("\"John\"".bytes().collect()));
-        let value = Expression::new("auth.identity.name").unwrap().eval();
-        assert_eq!(value, "John".into());
-        super::super::property::test::TEST_PROPERTY_VALUE.set(Some("-42".bytes().collect()));
+
+        property::test::TEST_PROPERTY_VALUE.set(Some((
+            property::Path::new(vec!["filter_state", "wasm.kuadrant.auth.identity.age"]),
+            "\"John\"".bytes().collect(),
+        )));
         let value = Expression::new("auth.identity.age").unwrap().eval();
+        assert_eq!(value, "John".into());
+
+        property::test::TEST_PROPERTY_VALUE.set(Some((
+            property::Path::new(vec!["filter_state", "wasm.kuadrant.auth.identity.name"]),
+            "-42".bytes().collect(),
+        )));
+        let value = Expression::new("auth.identity.name").unwrap().eval();
         assert_eq!(value, (-42).into());
+
```

Comment:
I guess it doesn't really matter as such as the path is somewhat irrelevant to the change but minor nit; age == John, name == -42

Reply:
I actually did it on purpose, to _not_ imply typing based of what otherwise could

Reply:
Ah, sure!